agreement/errors.go

```diff
@@ -16,38 +16,40 @@
 
 package agreement
 
-import "fmt"
+import (
+	"fmt"
+)
 
 // serializableError, or state machine error, is a serializable error that
 // is correctly written to cadaver files.
-type serializableErrorUnderlying string
-type serializableError = *serializableErrorUnderlying
+//msgp:ignore serializableErrorUnderlying
```

Comment:
this is weird. we call it `serializableError` but ignore from serialization. Plus, the ignore refers to serializableErrorUnderlying, not serializableError

Reply:
Yeah this was leftover from earlier iteration. Sorry